datasette/app.py

```diff
@@ -971,6 +973,7 @@ def inspect(self):
                 self._inspect[name] = {
                     'hash': m.hexdigest(),
                     'file': str(path),
+                    'size': get_size(path),
```

Comment:
I think the inspect mechanism should store the raw size here as an integer number of bytes. The KB/GB/MB display logic should be handled at display time, maybe as a custom template filter.